src/tags.json

```diff
@@ -95,6 +95,7 @@
   "moon": ["dark", "night"],
   "more-horizontal": ["ellipsis"],
   "more-vertical": ["ellipsis"],
+  "mouse-pointer": ["mouse", "pointer", "arrow", "cursor"],
```

Comment:
"mouse" and "pointer" are a little redundant here because they're already in the icon name.
```suggestion
  "mouse-pointer": ["arrow", "cursor"],
```

Reply:
ah ok. I wasn't sure if it would search independently. good to know for next time too!